roles/lib_openshift/src/test/unit/test_oc_route.py

```diff
@@ -250,6 +256,7 @@ def test_create_route(self, mock_cmd, mock_write, mock_tmpfile_copy, mock_oc_bin
         self.assertTrue(results['changed'])
         self.assertEqual(results['state'], 'present')
         self.assertEqual(results['results']['results'][0]['metadata']['name'], 'test')
+        self.assertEqual(results['results']['results'][0]['metadata']['labels']['route'], 'route')
```

Comment:
Do we need to update this to be more like this? https://github.com/openshift/openshift-ansible/blob/master/roles/lib_openshift/src/test/unit/test_oc_service.py#L396

Reply:
I think it should have the same effect, but I'm currently not even getting to that assert, it's just returning an empty list of results and it fails to find ['metadata']['name']